tasks/main.yml

```diff
@@ -113,28 +113,54 @@
     name: "{{ camac_releasedir }}/camac/django/"
     extra_args: "--upgrade"
 
+- name: reset django_migrations table
+  shell: |
+    psql \
+        -h {{ camac_db_host }} \
+        -p {{ camac_db_port }} \
+        -U {{ camac_db_username }} \
+        -d {{ camac_db_dbname }} \
+        -c "TRUNCATE django_migrations"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    PGPASSWORD: "{{ camac_db_password }}"
+
+- name: run camac fake migrations
+  shell: "python3 {{ camac_releasedir }}/camac/django/manage.py migrate --fake {{ item.app }} {{ item.migration }}"
+  when: camac_db_run_fake_migrations and camac_db_fake_migrations is defined
+  environment:
+    DJANGO_SETTINGS_MODULE: "camac.settings"
+    DJANGO_DATABASE_NAME: "{{ camac_db_dbname }}"
+    DJANGO_DATABASE_USER: "{{ camac_db_username }}"
+    DJANGO_DATABASE_PASSWORD: "{{ camac_db_password }}"
+    DJANGO_DATABASE_HOST: "{{ camac_db_host }}"
+    DJANGO_DATABASE_PORT: "{{ camac_db_port }}"
+    APPLICATION: "{{ camac_application }}"
```

Comment:
Why doesn't this have a DJANGO_ prefix?

I'm not sure if using the prefix is in vogue this week... IMO the situation is kinda like yarn vs. npm but for env variables in django settings :trollface:

Reply:
For historical reasons :smile: